test/unit/annotation_spec.js

```diff
@@ -231,6 +231,243 @@ describe('annotation', function() {
 
       expect(annotation.color).toEqual(new Uint8ClampedArray([0, 0, 0]));
     });
+
+    it('should parse Contents', function () {
+      const annotDict = new Dict();
+      annotDict.set('Contents', 'Content');
+
+      const annotation = new Annotation({ dict: annotDict, ref, });
+
+      expect(annotation.data.contents).toEqual('Content');
+    });
+  });
+
+  describe('MarkupAnnotation', function () {
+    it('should correctly parse T and not ' +
+      'parse IRT and RT when not defined',
+      function (done) {
+        const annotationRef = new Ref(820, 0);
+        const annotationDict = new Dict();
+        annotationDict.set('Type', Name.get('Annot'));
+        annotationDict.set('Subtype', Name.get('Text'));
+        annotationDict.set('T', 'Author Name');
+
+        const xref = new XRefMock([
+          { ref: annotationRef, data: annotationDict, },
+        ]);
+
+        AnnotationFactory.create(xref, annotationRef, pdfManagerMock,
+          idFactoryMock).then(({ data, }) => {
+            expect(data.title).toEqual('Author Name');
+
+            expect(data.inReplyTo).toBeUndefined();
+            expect(data.replyType).toBeUndefined();
+            done();
+          }, done.fail);
+      });
+
+  it('should correctly parse IRT and set default RT.',
```

Comment:
The indentation is a bit off here and in other parts of the added code in this file. This and the code below is part of the `describe` block and should therefore be indented with two extra spaces.